op-node/rollup/status/status.go

```diff
@@ -116,6 +117,7 @@ func (st *StatusTracker) OnEvent(ev event.Event) bool {
 	case rollup.ResetEvent:
```

Comment:
I suspect there may be some events missing from this switch such as `rollup.ForceResetEvent` https://github.com/ethereum-optimism/optimism/blob/a49e029a25c92c5567b0e84dd386b82aa44b06b6/op-node/rollup/event.go#L43-L49

If there were missing events, this might explain how the EL and CL can report very different heads in some situations.

Reply:
Yes I also suspect that we're missing catching some events here. It's worth going over all events that potentially update local safe and see if we're receiving those in the status tracker.

Reply:
Removed it [4e8461f](https://github.com/ethereum-optimism/optimism/pull/16296/commits/4e8461f5a5a0c04fd22124806771714bb68563e9)